### ISSUE TITLE
🐛 Bug Report: Unable to edit document after adding array attribute (Appwrite Cloud)

### ISSUE THREAD

**Issue by pluja** [appwrite/appwrite#5241]
### 👟 Reproduction steps

In Appwrite Cloud:

1. Create a collection, add some attributes (2 strings, for example).
2.  Add some documents to the collection.
3. Add a new string array attribute.
4. Try to edit existing documents from the UI.

### 👍 Expected behavior

I should be able to edit the attributes.

### 👎 Actual Behavior

I get a blank page and UI freezes. Developer console shows the following error:

```
Uncaught (in promise) TypeError: u[0][u[16].key] is undefined
    Immutable 51
    <anonymous> https://cloud.appwrite.io/console/project-6xxx8/databases/database-6xxx7
attributeForm-37c99cf9.js:1:15587
    Immutable 7
    InterpretGeneratorResume self-hosted:1822
    AsyncFunctionNext self-hosted:810
    Immutable 4
    <anonymous> https://cloud.appwrite.io/console/project-6xxx8/databases/database-6xxx7
```

### 🎲 Appwrite version

Version 1.1.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Appwrite Cloud Beta

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@pluja, thanks for creating this issue! 🙏🏼 I tried to follow your steps, but I couldn't reproduce the problem.

This document was created when the collection only had `firstString` and `secondString`

![Screen Shot 2023-03-17 at 9 34 54 AM](https://user-images.githubusercontent.com/1477010/225965061-9f9e19b1-cfb9-4546-843a-8b326686016b.png)

And I was able to update the document:

![Screen Shot 2023-03-17 at 9 35 16 AM](https://user-images.githubusercontent.com/1477010/225965184-76033340-040e-41dc-aab1-bb493b7fa6a6.png)

What could I be missing or doing wrong?
